README.md

```diff
@@ -111,6 +112,60 @@ module "rds_cluster_aurora_mysql" {
 }
 ```
 
+With [enhanced monitoring](examples/enhanced_monitoring)
+
+```hcl
+# create IAM role for monitoring
+resource "aws_iam_role" "iam_role" {
+  name               = "rds-${var.cluster-name}"
+  assume_role_policy = "${data.aws_iam_policy_document.policy_document.json}"
+}
+
+# attach amazon's managed policy for RDS to write logs
```

Comment:
Is this policy `to write logs`?
Probably should rename it to `Attach Amazon's managed policy for RDS enhanced monitoring`